CardParts/src/Classes/Card Parts/CardPartView.swift

```diff
@@ -20,15 +20,11 @@ public protocol CardPartView {
 extension CardPartView {
     
     public var view: UIView! {
-        get {
-            return self as! UIView
-        }
+        return (self as! UIView)
```

Comment:
I would get rid of implicitly unwrapped optional and have something like this.

```
public protocol CardPartView {
	...    
        var view : UIView { get }
	...
}
```

and

```
extension CardPartView where Self: UIView {   
    public var view: UIView {
            return self
    }
}
```

Reply:
Using implicitly unwrapped optionals and force casting would be crash-prone.